phpunit.xml.dist

```diff
@@ -8,14 +8,18 @@
     <directory suffix=".inc">.</directory>
     <directory suffix=".module">.</directory>
     <exclude>
+      <directory suffix="Drupal.php">.</directory>
       <directory suffix="Test.php">.</directory>
       <directory suffix=".api.php">.</directory>
       <directory suffix=".tpl.php">.</directory>
       <directory suffix=".features.inc">.</directory>
       <directory suffix=".features.field_base.inc">.</directory>
       <directory suffix=".features.field_instance.inc">.</directory>
+      <directory suffix=".features.filter.inc">.</directory>
       <directory suffix=".features.taxonomy.inc">.</directory>
       <directory suffix=".features.uuid_term.inc">.</directory>
+      <directory suffix=".field_group.inc">.</directory>
+      <directory suffix=".strongarm.inc">.</directory>
```

Comment:
Why not `.features.field_group.inc` and `.features.strongarm.inc` → would be more consistent with the other files?